src/main/java/org/sagebionetworks/web/client/SynapseJavascriptClient.java

```diff
@@ -407,36 +409,36 @@ private Request doDelete(String url, AsyncCallback callback) {
 		requestBuilder.configure(DELETE, url);
 		requestBuilder.setHeader(ACCEPT, APPLICATION_JSON_CHARSET_UTF8);
 		if (authController.isLoggedIn()) {
-			requestBuilder.setHeader(SESSION_TOKEN_HEADER, authController.getCurrentUserSessionToken());
```

Comment:
change the header that's used for authorization (since we're using an access token now).